go/Makefile

```diff
@@ -9,8 +9,6 @@ vet:
 
 lint:
 	golint ./... | grep -v ^vendor | grep -v ^protocol | grep -v comment | grep -v KeysById | grep -v "error should be the last type" | grep -v stutters || echo "Lint-free!"
-	# Strict linting on saltpack
-	golint saltpack/...
```

Comment:
Sounds good to me. I wonder if we should update the `test` target below to call our functioning `test.sh` script, while we're at it :) Or just delete it, I'd be fine with that too.